src/starpu/accumulate.cc

```diff
@@ -75,6 +75,23 @@ void init()
     codelet_fp32.modes[0] = static_cast<starpu_data_access_mode>(
             STARPU_RW | STARPU_COMMUTE);
     codelet_fp32.modes[1] = STARPU_R;
+
+    codelet_fp32_fast_tf32.init("nntile_accumulate_fp32_fast_tf32",
+            nullptr,
+            {cpu<fp32_t>},
+#ifdef NNTILE_USE_CUDA
+            {cuda<fp32_t>}
+#else // NNTILE_USE_CUDA
+            {}
+#endif // NNTILE_USE_CUDA
+            );
+    codelet_fp32_fast_tf32.nbuffers = 2;
+    codelet_fp32_fast_tf32.modes[0] = static_cast<starpu_data_access_mode>(
+            //STARPU_RW | STARPU_COMMUTE);
```

Comment:
Codelet allows A+BC+DE to be computed in (A+BC)+DE order and in (A+DE)+DC order. Uncomment line 90 and remove line 91.